src/NServiceBus.Core/Hosting/Helpers/AssemblyScannerConfiguration.cs

```diff
@@ -24,7 +24,7 @@ public class AssemblyScannerConfiguration
         /// <summary>
         /// Defines whether nested directories should be included in the assembly scanning process. Default value is <code>false</code>.
         /// </summary>
-        public bool ScanAssembliesInNestedDirectories { get; set; } = false;
+        public bool ScanAssembliesInNestedDirectories { get; set; }
```

Comment:
no it doesn't, but as a user if I'm interested in the default value, I'd immediately see the default value when looking at the source and probably trust this value to remain unless something is configured explicitly. As said, this is a personal opinion and I wouldn't mind to remove the defaults, but in that case it should be consistent across the file.